ReactNativeClient/lib/joplin-renderer/MdToHtml.ts

```diff
@@ -1,6 +1,7 @@
 import InMemoryCache from 'lib/InMemoryCache';
 import noteStyle from './noteStyle';
 import { fileExtension } from './pathUtils';
+import { ExtraContentScript } from 'lib/services/plugins/utils/loadContentScripts';
```

Comment:
The renderer shouldn't have a dependency to the lib as it could potentially be used separately (that has become clearer with the Lerna update). Perhaps you could simply leave the ExtraRendererRule interface in this file since it's compatible with ExtraContentScript.

Reply:
I'll keep that in mind going forward, thanks. This is fixed here.